jest.config.json

```diff
@@ -4,7 +4,7 @@
     "^.+\\.jsx?$": "babel-jest"
   },
   "moduleNameMapper": {
-    "\\.(png|svg)$": "<rootDir>/config/assetsTransformer.js",    
+    "\\.(png|svg)$": "<rootDir>/assetsTransformer.js",
```

Comment:
better to rename to ``assets.transformer.js``

Reply:
seems good